mvlearn/datasets/base.py

```diff
@@ -44,6 +53,11 @@ def load_UCImultifeature(select_labeled="all"):
     if select_labeled == "all":
         select_labeled = range(10)
 
+    if not shuffle:
+        random_state = 1
+    elif random_state is None:
```

Comment:
I think that random_state being None for np.random.seed will invoke a random seeding without the need to randomly assign random_state. Effectively, `np.random.seed(None)` will yield a random seeding.